### PR TITLE
Vue Updates

### PR DESCRIPTION
**Pull request by gnarula**
Updated the project to use Vue CLI which indirectly updated webpack.

Also updated to use the latest version of kyber and cothority.

Tested end to end locally on Microsoft Edge.

### CODE REVIEW

File: evoting/frontend/README.md [hunk 1]
```diff
@@ -1,30 +1,24 @@
...
-npm run build --report
-
-# run unit tests
-npm run unit
-
-# run e2e tests
-npm run e2e
-
-# run all tests
-npm test
```
Comment (pierluca) [inline evoting/frontend/README.md:27]: have the tests been removed?

File: evoting/frontend/package.json [hunk 2]
```diff
@@ -1,111 +1,61 @@
...
-  "description": "A Vue.js project",
-  "author": "DEDIS",
+  "version": "0.1.1",
   "private": true,
   "scripts": {
-    "dev": "genversion src/version.js && webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
-    "start": "npm run dev",
-    "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
-    "e2e": "node test/e2e/runner.js",
-    "test": "npm run unit && npm run e2e",
```
Comment (pierluca) [inline evoting/frontend/package.json:12]: has the code for the tests been removed, or only its running? 
it'd be good to still be able to run them, especially given the changes :)

File: evoting/frontend/package.json [hunk 3]
```diff
@@ -1,111 +1,61 @@
 {
   "name": "evoting",
-  "version": "1.10.0",
-  "description": "A Vue.js project",
-  "author": "DEDIS",
+  "version": "0.1.1",
```
Comment (pierluca) [inline evoting/frontend/package.json:3]: probably 1.11.0 ? 
it seems odd to decrement the version number :)

File: evoting/frontend/src/App.vue [hunk 4]
```diff
@@ -15,132 +15,133 @@
...
+/* .root-container {
   margin-top: 64px !important;
-}
-
-/* override - bug in vuetify https://github.com/vuetifyjs/vuetify/issues/3091 */
-.breadcrumbs li:last-child a {
-  cursor: pointer !important;
-  pointer-events: auto !important;
-}
+} */
```
Comment (pierluca) [inline evoting/frontend/src/App.vue:91]: dead code?

**Review by pierluca — COMMENTED** [review]
👍  LGTM, barring very minor remarks.
Feel free to ignore the comments about tests - visibly there were virtually none.

File: evoting/frontend/README.md (hunk 1 above)
Comment (gnarula) [reply evoting/frontend/README.md:27]: They didn't exist :)

File: evoting/frontend/package.json (hunk 3 above)
Comment (gnarula) [reply evoting/frontend/package.json:3]: Ah, that's the default one. I'll bump that one again

**Pull request merged**